src/dvclive/live.py

```diff
@@ -22,9 +23,13 @@
 logger = logging.getLogger("dvclive")
 logger.setLevel(os.getenv(env.DVCLIVE_LOGLEVEL, "INFO").upper())
 
+ParamsLike = Union[int, float, str, bool]
```

Comment:
❤️ . Would be awesome to also add this to the other methods `log`, `log_plot` `log_image` (not as part of this P.R)